cfgov/preprocessed/css/molecules/call-to-action.less

```diff
@@ -0,0 +1,41 @@
+/* topdoc
+  name: Call to Action
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+		<div class="m-call-to-action">
+		    <p class="short-desc">
+		    </p>
+	        <a class="btn btn__full m-call-to-action_btn" href="/">
```

Comment:
MINOR: Indent the `a` so that it fits with the Structural cheat sheet shown on L:15.